CRM/Financial/Form/FrontEndPaymentFormTrait.php

```diff
@@ -106,4 +106,30 @@ protected function getProcessors(): array {
     return $pps;
   }
 
+  /**
+   * Adds in either a set of radio buttons or hidden fields to contain the payment processors on a front end form
+   */
+  protected function addPaymentProcessorFieldsToForm() {
+    $paymentProcessors = $this->getProcessors();
+    $optAttributes = [];
+    foreach ($pamymentProcessors as $ppKey => $ppval) {
+      if ($ppKey > 0) {
+        $optAttributes[$ppKey]['class'] = 'payment_processor_' . strtolower($this->_paymentProcessors[$ppKey]['payment_processor_type']);
+      }
+      else {
+        $optAttributes[$ppKey]['class'] = 'payment_processor_paylater';
+      }
+    }
+    if (count($pamymentProcessors) > 1) {
+      $this->addRadio('payment_processor_id', ts('Payment Method'), $pamymentProcessors,
```

Comment:
@seamuslee001 Typo should be `$paymentProcessors`

Reply:
Thanks @mattwire fixed now